test/e2e/tests/ppom/security-alerts-api/security-alerts-api-trade-order-farming.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { MockttpServer } from 'mockttp';
+import { SECURITY_ALERTS_PROD_API_BASE_URL } from '../constants';
+import FixtureBuilder from '../../../fixture-builder';
+import {
+  WINDOW_TITLES,
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../../helpers';
+import { SecurityAlertResponse } from '../../../../../app/scripts/lib/ppom/types';
+
+const maliciousTradeAlert: SecurityAlertResponse = {
+  block: 1,
+  result_type: 'Malicious',
+  reason: 'trade_order_farming',
+  description: 'If you approve this request, you might lose your assets.',
+  features: [],
+};
+
+async function mockRequest(
+  server: MockttpServer,
+  response: SecurityAlertResponse,
+): Promise<void> {
+  await server
+    .forPost(`${SECURITY_ALERTS_PROD_API_BASE_URL}/validate/0x1`)
+    .withJsonBodyIncluding({
+      method: 'eth_sendTransaction',
+    })
+    .thenJson(201, response);
+}
+
+async function mockMaliciousResponses(
+  mockServer: MockttpServer,
+): Promise<void> {
+  await mockRequest(mockServer, maliciousTradeAlert);
+}
+
+describe('Security Alerts API - Set Trade farming order @no-mmi', function () {
```

Comment:
As many of these files have a single test in, would it be simpler to keep them in a single file to reduce the boilerplate overhead?

The duplication should and final line count should be minimal if we rely on page object models to abstract the interactions?